### PR TITLE
edit simple_bc documentation

### PR DESCRIPTION
**Pull request by saikrishna-1996**

### CODE REVIEW

**Comment by cloderic** [comment]
@saikrishna-1996 is it ready to review ?

**Comment by saikrishna-1996** [comment]
not yet..

File: docs/tutorials/simple_bc.md [hunk 1]
```diff
@@ -15,126 +15,114 @@ We train the agent online as trials are running and each trial uses the latest s
...
+python -m simple_mlflow
 ```
-and web client in another terminal:
+launch a simple behavior cloning run with the mountain car gym environment using:
 ```
-./run.sh web_client_start
+python -m main +experiment=simple_bc/mountain_car
 ```
 
 Open `localhost:8080`.
```
Comment (wduguay-air) [inline docs/tutorials/simple_bc.md:65]: Already mentioned in the paragraph at the bottom of this section

File: docs/tutorials/simple_bc.md [hunk 2]
```diff
@@ -15,126 +15,114 @@ We train the agent online as trials are running and each trial uses the latest s
...
-
-- simple_bc_cartpole
-- simple_bc_mountaincar
-- simple_bc_lander
 
 Start a training run as follows in the third terminal:
 
 ```
 ./run.sh client start simple_bc_lander
 ```
```
Comment (wduguay-air) [inline docs/tutorials/simple_bc.md:72]: deprecated in next branch

**Pull request merged**